src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessorTests.java

```diff
@@ -145,20 +146,20 @@ public boolean accept(final File dir, final String name) {
         new MemoryDBSnapshot(conn, MetricName.MASTER_PENDING, 6001L);
     long lastUpdateTime1 = 2000L;
     masterPendingSnap1.setLastUpdatedTime(lastUpdateTime1);
-    Object[][] values1 = {{0}};
+        Object[][] values1 = {{"delete-index", 0}};
```

Comment:
nit: can remove the indention?

Reply:
Please run `./gradlew spotlessApply` to format the code. That should solve the indentation issues.